src/api/map/routes/map.ts

```diff
@@ -0,0 +1,34 @@
+import { procedure, router } from "../../trpc";
+import { authUser } from "../../auth";
+
+const get = procedure
+  .use(authUser("Mentee"||"Mentor"))
+  .query(async () =>
+{
+});
+
+
+const list = procedure
```

Comment:
let's start with this single API function and remove all others. let's implement map reading end-to-end first before moving on to updating etc.